packages/webpack.config.js

```diff
@@ -2,9 +2,11 @@ const fs = require('fs');
 const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const ThreeMinifierPlugin = require('@yushijinhun/three-minifier-webpack');
```

Comment:
I thought this was not satisfactory (https://github.com/silx-kit/h5web/issues/216#issuecomment-700613125) ?

Reply:
It wasn't satisfactory in the sense that not much was being removed with tree-shaking (because of `drei`). It still makes the tree-shaking work for ThreeJS.